apps/platform-app/src/app/[locale]/(profile)/_actions/mutations/fetchCVurl.ts

```diff
@@ -0,0 +1,11 @@
+'use server'
+
```

Comment:
**nit:** This is query not mutation. Btw, I think we can remove this anyway, as CV URL is included in ProfileModel that is fetched when initializing form. Such url should be available in form values object.